From 97b2a41ea7b1e628601b87e922685b659edb6272 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Thu, 16 Jun 2016 15:45:25 +0800
Subject: [PATCH] ARM64: dts: rk3399-evb: add keep-power-in-suspend for sdhci

We not ONLY keep card power despite of the power status of Soc,
so we should add this tag to speed up the wake.

Change-Id: I318342829cf2f5ef6d3e250c59979c156b412123
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
index b9243395e2e2..7e3f588cc088 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb.dtsi
@@ -261,6 +261,7 @@
 	mmc-hs400-1_8v;
 	supports-emmc;
 	non-removable;
+	keep-power-in-suspend;
 	mmc-hs400-enhanced-strobe;
 	status = "okay";
 };
-- 
2.35.3

